extensions/aerospace/assets/scripts/list-windows.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# References:
+# - https://nikitabobko.github.io/AeroSpace/commands#list-windows
+# - https://www.alfredapp.com/help/workflows/inputs/script-filter/json/
+
+result=$(aerospace list-windows --workspace visible --format "%{app-name},%{window-title},%{window-id},%{app-pid},%{workspace},%{app-bundle-id}")
```

Comment:
I don't have control over this PR, and the final say is after people who support the raycast plugin. 

But as a maintainer of AeroSpace, the hacking around unstructured format makes me sad. Sure, you can use it for your own needs, but when you publish it for other people, I think it's better to clean it up. I'd wait for https://github.com/nikitabobko/AeroSpace/issues/577 before merging this PR

I want to take this opportunity to remind that the plugin currently parses `aerospace.toml` config itself (which is fragile), while the proper API was provided https://github.com/nikitabobko/AeroSpace/issues/215#issuecomment-2119209902 (and still waits to be used)

This PR increases the technical debt of the raycast plugin

Reply:
List-windows.sh shouldn’t have been added. The command uses array-windows.sh but your comment still applies.
I don’t mind waiting if it will improve the code and technical debt. I started the PR before seeing the —json improvement.